### DISCUSSION TITLE
Select method fromModel with a large number of records

### DISCUSSION THREAD

**Discussion by RuLeiFei** [orchidsoftware/platform#2734] [Q&A]
Hello everyone, I'm testing this awesome package for my work. I studied the documentation and chose the fromModel method to search for data from the source. 

`Select::make('region_id')->fromModel(Region::class, 'title', 'id')->empty('Empty')->title('Region')`

I have 15,000 Person records in my working database and this number will grow. I generated 50,000 records with a factory for a test project, and with many records, the interface reloaded, and the search was slow. There is some method for accessing the server, for example method `Like ` or another solution, AJAX etc.

Thanks for any help!

**Comment by tabuna** [comment]
Hi @RuLeiFei You are looking for the `Relation` field, it loads data dynamically.
https://orchid.software/en/docs/field/#relation

**Reply by RuLeiFei** [reply]
Thanks a lot, awesome package! I should spend more time going through the documentation!
